### PR TITLE
fix: add rbac with resource roles and domain example

### PR DESCRIPTION
**Pull request by selfuppen**
Fix: https://github.com/casbin/jcasbin/issues/328

test: add rbac with resource roles and domain example

### CODE REVIEW

**Comment by hsluoyz** [comment]
@longxu0509

File: src/test/java/org/casbin/jcasbin/main/RbacAPIWithDomainsUnitTest.java [hunk 1]
```diff
@@ -98,4 +98,28 @@ public void testImplicitPermissionAPIInDomain() {
 
         testGetImplicitPermissionsInDomain(e, "alice", "domain1", asList(asList("admin", "domain1", "data1", "read"), asList("admin", "domain1", "data1", "write")));
     }
+    @Test
```
Comment (longxu0509) [inline src/test/java/org/casbin/jcasbin/main/RbacAPIWithDomainsUnitTest.java:101]: Test cases should be added to the src/test/java/org/casbin/jcasbin/main/ModelUnitTest.java.

**Review by longxu0509 — COMMENTED** [review]
Test cases should be added to the src/test/java/org/casbin/jcasbin/main/ModelUnitTest.java.

**Comment by hsluoyz** [comment]
@selfuppen

**Review by longxu0509 — COMMENTED** [review]
LGTM

**Pull request merged**